src/Components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo/UserInfo';
+
+import { Todo } from '../Types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
```

Comment:
You can do this without writing 'return'
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => (
```